.circleci/config.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       - run:
           name: Build Valhalla Docker Images
           command: |
+            docker pull ubuntu:20.04
             docker build -f ./docker/Dockerfile-build --tag valhalla/valhalla:build-latest ./docker
```

Comment:
imho:  the `docker build --pull -f ./docker/Dockerfile-build --tag valhalla/valhalla:build-latest ./docker` is better/safer 
see: https://docs.docker.com/engine/reference/commandline/build/
 * `--pull |   | Always attempt to pull a newer version of the image`

Reply:
jup agreed, didn't know that flag.. it doesn't specifically say "pull base image" but anything else would be weird;)

Reply:
:-)  not critical suggestions ... 
in the `docker-postgis` repo - this is already used: 
https://github.com/postgis/docker-postgis/blob/dabd4d3824cf32e633cf2a6f07515e5e13bc3774/Makefile#L79

Reply:
done